website/src/lib/oasst_api_client.ts

```diff
@@ -3,16 +3,10 @@ import { LeaderboardReply, LeaderboardTimeFrame } from "src/types/Leaderboard";
 import type { AvailableTasks } from "src/types/Task";
 import type { BackendUser, BackendUserCore, FetchUsersParams, FetchUsersResponse } from "src/types/Users";
 
-export class OasstError {
+export interface OasstError {
```

Comment:
changed to an interface since next js is not able serialize this object, it always shows `error [Object object]` in the console, which is not helpful.

when left as a normal object, nextjs per default will show all errors in the console

Reply:
I added a to string and it worked.